src/plugins/intel_cpu/src/nodes/lora.cpp

```diff
@@ -87,7 +93,10 @@ void LoRA::createPrimitive() {
 
     std::vector<MemoryPtr> inputMemory;
     for (size_t i = 0; i < getOriginalInputsNumber(); i++) {
-        inputMemory.emplace_back(getSrcMemoryAtPort(i));
+        auto srcEdgeMem = getSrcMemoryAtPort(i);
+        auto mem = std::make_shared<Memory>(getEngine(), srcEdgeMem->getDescPtr(), srcEdgeMem->getMemoryBlock());
```

Comment:
nit, don't we wnat to have clone method (or smt like that) for such purposes?

Reply:
Yes, it makes sense. But the real problem here is the fact that we must create a new memory object instead of reusing the existing one.